src/index_update_db.js

```diff
@@ -2,60 +2,177 @@
 
 const AWS = require('aws-sdk');
 const McawsModels = require('./models/mcawsModels.js');
+let mcawsDbObj = null;
+
+function addDradminRole(dbAwsAccount, drAdminRole,
+                        accountDetails, fullAdminRoleName) {
+  return new Promise((resolve, reject) => {
+    if(dbAwsAccount.account_type.toLowerCase() != 'craws') {
+      console.log("Account type is not craws. Hence skipping.");
+      resolve(true);
+    } else {
+      mcawsDbObj.Role(roleResp => {
+        roleResp.findOne({
+          where: drAdminRole
+        })
+        .then(roleData => {
+          if(roleData) {
+            return roleData;
+          } else {
+            console.log("dradmin role name provided does not exists");
+            reject("dradmin role name error");
```

Comment:
should the step be failing if the dradmin group is not found?

Reply:
we are not failing the step we are catching this rejection and proceeding with other steps